en/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: news_post
+title: "Support plans for Ruby 2.0.0 and Ruby 2.1"
+author: "usa"
+translator:
+date: 2016-02-24 09:00:00 +0000
+lang: en
+---
+
+We announce the future support plans for Ruby 2.0.0 and Ruby 2.1.
+
+## About Ruby 2.0.0
+
+As it has been announced before, all support for Ruby 2.0.0 has ended today.
+Bug and security fixes from more recent Ruby versions will no longer be backported to 2.0.0, and no further patch release of 2.0.0 will be released.
+
+We highly recommend that you upgrade to Ruby 2.3 or 2.2 as soon as possible.
+
+Please contact us via the ruby-core ML if you'd like to continue maintaining the 2.0.0 branch for some justifiable reason you can't upgrade.
+
+## About Ruby 2.1
+
+We are planning to release Ruby 2.1.9 until the end of March.
```

Comment:
I think

> *by* the end of March.

is the best option. "Before" is also valid. The problem with "until" is that it's more associated with continuous actions (just like in Ruby), such as "I'll be in Brisbane until the end of March".